site/code/index.html.js

```diff
@@ -305,6 +305,7 @@ Apollo Server also supports all Node.js HTTP server frameworks: Express, Connect
 
   - [graphql-php](https://github.com/webonyx/graphql-php): A PHP port of GraphQL reference implementation
   - [graphql-relay-php](https://github.com/ivome/graphql-relay-php): A library to help construct a graphql-php server supporting react-relay.
+  - [lighthouse](https://github.com/nuwave/lighthouse): GraphQL Server for Laravel
```

Comment:
It is Lighthouse with a capital L :)

```suggestion
  - [Lighthouse](https://github.com/nuwave/lighthouse): GraphQL Server for Laravel
```